### PR TITLE
creating model for csv bean

### PR DESCRIPTION
**Pull request by truhacevkir**
- Creating an object for csv entries from csv we download from channel register. 
- Following bean will be used to fetch entries from channel register.

### CODE REVIEW

**Comment by truhacevkir** [comment]
Første utkast av modellen, vi får se hvilke felter vi har bruk for etterhvert.

File: src/main/java/no/sikt/nva/pubchannels/channelregistrycache/ChannelRegistryCacheEntry.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
+package no.sikt.nva.pubchannels.channelregistrycache;
+
+import static nva.commons.core.attempt.Try.attempt;
+import com.opencsv.bean.CsvBindByName;
+import java.util.Arrays;
+import java.util.List;
+import no.unit.nva.commons.json.JsonUtils;
+import nva.commons.core.JacocoGenerated;
+
+public class ChannelRegistryCacheEntry {
```
Comment (monaullah) [inline src/main/java/no/sikt/nva/pubchannels/channelregistrycache/ChannelRegistryCacheEntry.java:10]: Why add data we don't use? You can remove "DOAJ", "Publiseringsavtale", "Internasjonal tittel" and "regDato"

Comment (truhacevkir) [reply src/main/java/no/sikt/nva/pubchannels/channelregistrycache/ChannelRegistryCacheEntry.java:10]: I do not know yet what we will be using, but if you sure, I remove it.

Comment (monaullah) [reply src/main/java/no/sikt/nva/pubchannels/channelregistrycache/ChannelRegistryCacheEntry.java:10]: You can see what we use today in `no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistryJournal`, `no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistryPublisher` and `no.sikt.nva.pubchannels.channelregistry.model.ChannelRegistrySeries`

**Review by monaullah — APPROVED** [review]

**Pull request merged**